spec/requests/manifests_search_spec.rb

```diff
@@ -8,5 +8,11 @@
       get "/manifests?aq[content.generator.name]=company"
       expect(response.body).to include(manifest.tracking_number.to_s)
     end
+
+    it 'paginates' do
+      10.times { create(:manifest, :indexed) }
```

Comment:
what about creating 5 records? that would test that pagination works just as well as creating 10 records, wouldn't it?